lib/src/const.nr

```diff
@@ -16,5 +20,8 @@ global MAX_STORAGE_VALUE_LENGTH: u64 = 32;
 /// Maximum number of bytes in an account state
 global MAX_ACCOUNT_STATE_LENGTH: u64 = 134;
 
+// TODO: make it generic
+global MAX_RECEIPT_RLP_LENGTH: u64 = 525; // TODO: Understand why
```

Comment:
I think we can skip on the second comment if we already have the first.

Reply:
This should be `MAX_RECEIPT_RLP_LENGTH` as it's RLP + tx type